webviz_subsurface/plugins/_group_tree/_views/_group_tree_view/_view.py

```diff
@@ -0,0 +1,350 @@
+from enum import Enum
+from typing import Any, Dict, List, Optional, Tuple
+
+import webviz_core_components as wcc
+import webviz_subsurface_components as wsc
+from dash import Input, Output, State, callback, html
+from webviz_config.webviz_plugin_subclasses import SettingsGroupABC, ViewABC
+
+from ..._types import NodeType, StatOptions, TreeModeOptions
+from ..._utils import EnsembleGroupTreeData
+from ._view_element import GroupTreeViewElement
+
+
+class ViewControls(SettingsGroupABC):
+    class Ids(str, Enum):
+        TOUR_STEP = "tour-step"
+        ENSEMBLE = "ensemble"
+        TREEMODE = "tree-mode"
+
+    def __init__(self, ensembles: List[str]) -> None:
+        super().__init__("Controls")
+        self._ensembles = ensembles
+
+    def layout(self) -> html.Div:
+        return html.Div(
+            id=self.register_component_unique_id(self.Ids.TOUR_STEP),
+            children=[
```

Comment:
You don't need to have this outer div. Remove it and just return the list of children. In the tour steps, you can use the `unique_id` of the settings group.

```python
"id": self.view(PluginIds.Views.GROUPTREE_VIEW).settings_group(GroupTreeView.Ids.FILTERS).get_unique_id()
```